packages/core/package.json

```diff
@@ -488,6 +488,7 @@
         "bytes": "^3.1.2",
         "cross-fetch": "^4.0.0",
         "cross-spawn": "^7.0.3",
+        "diff": "^7.0.0",
```

Comment:
we technically have this as a dev dependency right now btw. If we want to promoted it to a regular dependency we should remove it from there

Reply:
I have moved this to dependencies, also keep the version as `^5.1.0` to avoid breaking existing functionalities.